crates/ocg_common/src/voxel/biomes.rs

```diff
@@ -48,7 +57,9 @@ pub fn setup_basic_biomes(biome_registry: &mut BiomeRegistry) {
                 let (i_grass, _) = block_registry.lookup_name_to_object(GRASS_BLOCK_NAME.as_ref()).unwrap();
                 let (i_dirt, _) = block_registry.lookup_name_to_object(DIRT_BLOCK_NAME.as_ref()).unwrap();
                 let (i_stone, _) = block_registry.lookup_name_to_object(STONE_BLOCK_NAME.as_ref()).unwrap();
-                let (i_snow_grass, _) = block_registry.lookup_name_to_object(SNOWY_GRASS_BLOCK_NAME.as_ref()).unwrap();
+                let (i_snow_grass, _) = block_registry
+                    .lookup_name_to_object(SNOWY_GRASS_BLOCK_NAME.as_ref())
+                    .unwrap();
```

Comment:
why different?

Reply:
It overflowed the line length limit, `cargo fmt` updated the formatting everywhere. It's at 120 and I'd rather not make it longer